src/Altinn.Apps/AppFrontend/react/shared/src/language/texts/en.ts

```diff
@@ -74,6 +74,7 @@ export function en() {
       action: 'Action',
       add: 'Add',
       add_connection: 'Add connection',
+      add_new: 'Legg til ny',
```

Comment:
Norwegian text in file for English texts.

Reply:
Thanks, good catch. Fixed :)